GTViewController/Classes/Custom Classes/GTFollowupModalView.m

```diff
@@ -115,7 +97,27 @@ - (instancetype)initFromElement:(TBXMLElement *)element withStyle:(GTPageStyle*)
                 currentY += buttonPairView.frame.size.height + 10;
                 
             }else if ([modalComponentElementName isEqual:kName_Thank_You]) {
-                self.thankYouPage = nil;
+                
```

Comment:
I agree it's long, but I don't see an obvious way to split it up.  Most of the sections that theoretically could be split out would require passing a bunch of parameters and/or creating class member instances for things that don't need to be scoped beyond a single method call.